spec/googleauth/service_account_spec.rb

```diff
@@ -169,14 +173,33 @@ def cred_json_text
     it_behaves_like "jwt header auth"
   end
 
-  context "when enable_self_signed_jwt is set" do
+  context "when enable_self_signed_jwt is set with aud" do
     before :example do
+      @client.scope = nil
       @client.instance_variable_set(:@enable_self_signed_jwt, true)
     end
 
     it_behaves_like "jwt header auth"
   end
 
+  context "when enable_self_signed_jwt is set with scope" do
+    before :example do
+      @client.scope = ['scope/1', 'scope/2']
+      @client.instance_variable_set(:@enable_self_signed_jwt, true)
+    end
+
+    it_behaves_like "jwt header auth", nil
+  end
+
+  describe "#apply!" do
+    it "should raise error when scope and aud are both specified" do
```

Comment:
It does look like you have to update this test.

Reply:
Done!